File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.py
N = int(input())
for i in range(1,N) :
  print(" " * (N - i),end="")
  print('*' * (i + (i - 1)))

for i in range(N,0,-1) :
  print(" " * (N - i),end="")
  print('*' * (i + (i - 1)))

File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().split("\n");

let number = parseInt(input[0]);

for (let i = 1; i < number; i++) {
  let blank = " ".repeat(number - i);
  let stars = "*".repeat(i + (i - 1));
  console.log(blank + stars);
}
for (let j = number; j > 0; j--) {
  let blank = " ".repeat(number - j);
  let stars = "*".repeat(j + (j - 1));
  console.log(blank + stars);
}
